src/services/module_service.ts

```diff
@@ -2,8 +2,13 @@ import IModule from "../interfaces/module.ts";
 import { colours, LoggerService } from "../../deps.ts";
 import DenoService from "../services/deno_service.ts";
 import NestService from "../services/nest_service.ts";
+import GitHubService from "../services/github_service.ts";
 
-const supportedUrls = ["https://deno.land/", "https://x.nest.land"];
+const supportedUrls = [
+  "https://deno.land/",
```

Comment:
can we add a comment as to why deno.land requires a trailing slash and the others don't please

Reply:
Tbf i dont think it does, but you are right, it's probably just confusing, will change it 👍

Reply:
https://github.com/drashland/dmm/pull/91/commits/0bde1b214e9e42bbd7be9ad86e5303b3be1d2a2a